routers/api/v1/org/team.go

```diff
@@ -10,11 +10,16 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/context"
 	"code.gitea.io/gitea/routers/api/v1/convert"
+	"code.gitea.io/gitea/routers/api/v1/user"
 )
 
 // ListTeams list all the teams of an organization
 func ListTeams(ctx *context.APIContext) {
 	org := ctx.Org.Organization
+	if !org.IsOrgMember(ctx.User.ID) {
```

Comment:
I don't understand what you are saying; this is an endpoint for viewing an organization's teams, not for viewing the members of those teams.

Regardless, the corresponding endpoint in the Github API (`GET /orgs/:orgname/teams`) is only accessible to organization members, so I believe that is what we want to do here